src/Form/Resolver/Choice.php

```diff
@@ -146,6 +146,7 @@ private function getEntityValues(Bag $field)
 
         $filter = $field->get('filter');
         $filter['order'] = $field->get('sort');
+        $filter['limit'] = $field->get('limit') ?? 500;
```

Comment:
$field->get('limit') has 500 as default value so ` ?? 500` is useless here.

Reply:
@JarJak I have removed the coalesce (because it is not compatible with PHP 5.5.9) and kept 500 as default value. In my tests, I saw no limit in both cases, returning a very large result set.

With this change, the documentation will now have this information as true. :smile:

Reply:
Great! Can you set 500 as `DEFAULT_LIMIT` const?